include/spl/sys/debug.h

```diff
@@ -107,13 +144,13 @@ void spl_dumpstack(void);
  */
 #else
 
-#define	ASSERT(cond)		VERIFY(cond)
+#define	ASSERT3B VERIFY3B
+#define	ASSERT3S VERIFY3S
+#define	ASSERT3U VERIFY3U
+#define	ASSERT3P VERIFY3P
+#define	ASSERT0 VERIFY0
+#define	ASSERT VERIFY
 #define	ASSERTV(x)		x
```

Comment:
Food for thought, we could retire `ASSERTV`.  In nearly all instances it could be replaced by `__attribute__ ((unused))` which is understood by both `gcc` and `llvm`.